module/owlib/src/include/ow_infernoembedded.h

```diff
@@ -0,0 +1,21 @@
+/*
+$Id$
```

Comment:
CVS keywords where removed in 52b1059a5bc7efb6ac8891e7c2b90433d0bdd0f4 
No need to have one here.

Reply:
Ok